erts/emulator/zstd/vendor.info

```diff
@@ -0,0 +1,13 @@
+{
+  "ID": "erts-zstd",
+  "description": "zstd",
+  "copyrightText": "Copyright (c) Meta Platforms, Inc. and affiliates. All rights reserved.",
+  "downloadLocation": "https://github.com/facebook/zstd",
+  "homepage": "https://github.com/facebook/zstd",
+  "license": "BSD-3-Clause OR GPL-2.0-only",
+  "licenseDeclared": "BSD-3-Clause OR GPL-2.0-only",
+  "name": "zstd",
+  "versionInfo": "v1.5.6",
+  "path": "./erts/emulator/zstd",
```

Comment:
Some files in the zstd directory are Apache-2.0 that I have created. They are scripts and makefiles things. Do they need to be handled somehow?

Reply:
No, I do not think so. This is the zstd library as such and as declared by the owner.
We have done changes and added files, as long as those files have their SPDX or header, this is fine.

the `vendor.info` simply lists the origin of the vendor library and the license of the library, but inside the library, there may be other (compatible) licenses in it

(this is my understanding)

Reply:
Great!